sample-apps/react-native/dogfood/src/components/ParticipantActions.tsx

```diff
@@ -1,6 +1,6 @@
 import {
   Avatar,
-  colorPallet,
+  colorPalette,
```

Comment:
As this is anyway a breaking change (renamed 😅 ), can we remove this export completely? `useTheme()` should be the way to go here.